### PR TITLE
DENG 1051 Shredder Prototype

### PR DESCRIPTION
**Pull request by lucia-vargas-a**
[DENG-1051](https://mozilla-hub.atlassian.net/browse/DENG-1051). Collect KPIS based on deletion requests for Firefox Desktop.
 

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

[DENG-1051]: https://mozilla-hub.atlassian.net/browse/DENG-1051?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-2163)

### CODE REVIEW

File: sql_generators/active_users_deletion_requests/__init__.py [hunk 1]
```diff
@@ -50,23 +51,41 @@ def generate(target_project, output_dir, use_cloud_function):
...
-            sql=reformat(
-                mobile_deletion_request_query_template.render(
-                    project_id=target_project,
-                    app_value=browser.value,
-                    app_name=browser.name,
-                )
-            ),
-            skip_existing=False,
-        )
+        if browser.name == "firefox_desktop":
```
Comment (kik-kik) [inline sql_generators/active_users_deletion_requests/__init__.py:61]: Why the condition here? It both blocks appear to be the same?

Comment (kik-kik) [reply sql_generators/active_users_deletion_requests/__init__.py:61]: Never mind, I just noticed the template used is different.

File: sql_generators/active_users_deletion_requests/templates/desktop_deletion_request_query.sql
Comment (kik-kik) [inline sql_generators/active_users_deletion_requests/templates/desktop_deletion_request_query.sql:1]: According to the metadata template looks like we expect to have a `partition_date` field which is used for partitioning the table but it appears to be missing in this query?

https://github.com/mozilla/bigquery-etl/blob/76d87c1638984477d3e3cfc40a30f33f22ea2c17/sql_generators/active_users_deletion_requests/templates/metadata_deletion_request.yaml#L25

**Review by kik-kik — APPROVED** [review]
Looks like an extension of the existing solution.

**Review by relud — APPROVED** [review]

**Pull request merged**